src/components/App/App.module.css

```diff
@@ -1,8 +1,24 @@
+body {
+  background-color: #fce4ec;
+}
+
 .wrap {
-	margin: auto;
-	width: 650px;
+  display: flex;
+  justify-content: space-between;
+  margin: auto;
+  width: 800px;
+  padding-top: 10px;
+}
+
+.link {
+  text-decoration: none;
+  color: black;
 }
 
-.title {
-	font-family: 'Georgia';
+.sidebar {
+  width: 150px;	
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/TODO-Olga/pullRequest?prid=5555083)